Stripe/STPAddressViewModel.m

```diff
@@ -224,6 +224,7 @@ - (void)setAddress:(STPAddress *)address {
                 break;
             case STPAddressFieldTypeCountry:
                 cell.contents = address.country;
+                [cell delegateCountryCodeDidChange:address.country];
```

Comment:
ah yup, good point, I was a bit too quick to jump to this as a fix. Turns out the issue was actually in `setContents` – previously, we were setting the country field's text to the country code, instead of getting the name from the country picker.